From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Fri, 3 May 2024 16:47:57 +1000
Subject: disable-qtexampleicons

---
 sources/pyside6/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pyside6/sources/pyside6/CMakeLists.txt
===================================================================
--- pyside6.orig/sources/pyside6/CMakeLists.txt
+++ pyside6/sources/pyside6/CMakeLists.txt
@@ -42,4 +42,4 @@ endif()
 
 add_subdirectory(doc)
 
-add_subdirectory(qtexampleicons)
+#add_subdirectory(qtexampleicons)
